### PR TITLE
ChangePassword : create change password / forgot password component

### PR DESCRIPTION
**Pull request by deveshk1**
Create Screen for Forgot password
Create screen for change password
Connect with backend API
add necessary messages for user

### CODE REVIEW

File: middlewares/auth/index.ts [hunk 1]
```diff
@@ -278,8 +292,79 @@ export function createAuthMiddleware(
...
+            link: link,
+            linkExp: Date.now()+10*60*1000,
+            secret
+        }
+         await db.insert(TABLE_FORGOTPASSWORD,emailObj)
+         res.send(ApiResponse.ok("If you are registered with us , an email will be sent to reset the password "))
+
+         //send email
+         console.log(emailObj.link)
+        //  config.mailer?.sendTextEmail(email,"Reset Password",emailObj.link)
```
Comment (shiveshnavin) [inline middlewares/auth/index.ts]: uncomment

File: middlewares/auth/mail/email_template.html [hunk 2]
```diff
@@ -14,19 +14,19 @@ <h2>{{ appname }}</h2>
 
         < !--- Email body-- ->
```
Comment (shiveshnavin) [inline middlewares/auth/mail/email_template.html:15]: fix template

**Comment by deveshk1** [comment]
image
image

**Review by shiveshnavin — APPROVED** [review]
LGTM

**Pull request merged**